app/test_main.py

```diff
@@ -1,3 +1,29 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected_result",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17]),
+        ],
+    )
+    def test_human_age_correctly(
+            self,
+            cat_age: int,
+            dog_age: int,
+            expected_result: list[int]
+    ) -> None:
+        assert get_human_age(cat_age, dog_age) == expected_result, (
+            f"For cat age {cat_age} and dog age {dog_age} "
+            f"result should be {expected_result}"
+        )
```

Comment:
Since you cover corner cases, I suggest to try more detalized messages as 1 more parameter